README.md

```diff
@@ -50,14 +51,68 @@ git clone https://github.com/smartcontractkit/foundry-starter-kit
 cd foundry-starter-kit
 ```
 
-## Install dependencies as follows:
+## Install dependencies as follows: 
 
 Run `forge install` to install dependencies. [Foundry uses git submodules](https://book.getfoundry.sh/projects/dependencies) as its dependency management system.
 
-> ⚠️  when running forge install, you may see an error message if you have uncomitted changes in your repo.  Read the message carefully - it may inform you that you can add the `--no-commit` flag to each of these `install` commands if your workspace has uncommitted changes.
+> ⚠️  when running forge install, you may see an error message if you have uncommitted changes in your repo.  Read the message carefully - it may inform you that you can add the `--no-commit` flag to each of these `install` commands if your workspace has uncommitted changes.
 
 You can update dependencies by running `forge update`
 
+## Installing Chainlink contracts with `forge install`:
+
+> [!IMPORTANT]
+> For this starter kit, the packages have already been included and can be installed using the above `forge install` command. This package includes Chainlink contracts v1.5.0. The below steps are for those that wish to install Chainlink contracts into their own project or modify the version used within the starter kit.
+
+<details id=">=1.5.0">
+<summary>v1.5.0 and above</summary>
+
+### Step 1: Install >= v1.5.0 and required dependencies
+
```

Comment:
Maybe add a comment with something along the lines of that this is how you would add Chainlink v1.5.0 contracts to an existing Foundry project, but not this repo due to the blurb above.